src/grisp_cryptoauth.app.src

```diff
@@ -1,6 +1,6 @@
 {application, grisp_cryptoauth,
  [{description, "Secure Element support for GRiSP2 based on cryptoauthlib"},
-  {vsn, "2.3.0"},
+  {vsn, "2.3.1"},
```

Comment:
Please, do not bump the version in this PR, we should follow the standard procedure and commit version changes during the release process. BTW, I would do a 2.4.0

Reply:
@ziopio done